### ISSUE TITLE
Not precise statement for those two columns of `git status -s`

### ISSUE THREAD

**Issue by YueLinHo** [progit/progit2#557]
book\02-git-basics\sections\recording-changes.asc line 187
```
There are two columns to the output - 
the left hand column indicates that the file is staged
and the right hand column indicates that it's modified.
```

According [git status man](https://git-scm.com/docs/git-status) (`X`: Left, `Y`: Right)
```
X shows the status of the index
Y shows the status of the work tree.
```

and other infor:
```
For untracked paths, XY are ??. 
Other status codes can be interpreted as follows:

' ' = unmodified
M = modified
A = added
D = deleted
R = renamed
C = copied
U = updated but unmerged

Ignored files are not listed, unless --ignored option is in effect, in which case XY are !!.

X          Y     Meaning
-------------------------------------------------
          [MD]   not updated
M        [ MD]   updated in index
A        [ MD]   added to index
D         [ M]   deleted from index
R        [ MD]   renamed in index
C        [ MD]   copied in index
[MARC]           index and work tree matches
[ MARC]     M    work tree changed since index
[ MARC]     D    deleted in work tree
-------------------------------------------------
D           D    unmerged, both deleted
A           U    unmerged, added by us
U           D    unmerged, deleted by them
U           A    unmerged, added by them
D           U    unmerged, deleted by us
A           A    unmerged, both added
U           U    unmerged, both modified
-------------------------------------------------
?           ?    untracked
!           !    ignored
-------------------------------------------------
```

**Comment by ben** [comment]
Yeah, this could use some updating.

**Comment by YueLinHo** [comment]
Is this good enough?
```
There are two columns to the output - 
the left hand column indicates the status of the staging area
and the right hand column indicates the status of the working tree.
```

**Comment by ben** [comment]
Yeah, I like it.

**Comment by YueLinHo** [comment]
Fixed with PR #558.
